tools/ExtensionLibrary/DevHome.ExtensionLibrary/TelemetryEvents/ReportInstalledExtensionEvent.cs

```diff
@@ -13,7 +13,7 @@ namespace DevHome.ExtensionLibrary.TelemetryEvents;
 [EventData]
 public class ReportInstalledExtensionEvent : EventBase
 {
-    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServiceUsage;
+    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServicePerformance;

```

Comment:
I'm not sure what this would count as -- we don't send telemetry when an extension is installed or uninstall, or enabled or disabled, since that could provide a lot of noise. Instead, IIRC, we send an event per extension on Dev Home startup. So technically we're not sending on a user action, but we are sending a user's choice. What should that count as?

Reply:
We're effectively sending data regardless of user direct action, whatever happens to be installed. But some things get auto-installed like the GitHub extension, right?  I would think this should stay the same and if we want to capture a user-install choice we create a new event for that and set it as ProductAndServiceUsage and tie it directly to a user action.